### PR TITLE
Idea experimental fetures

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
Add procedure to enable experimental features support for Idea plugin

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/tools-ides.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ You can test a Ballerina file or module using the following command:
 
 ## Compatibility
 
-Ballerina Language Specification supports a set of experimental features such as *_transactions_* syntax. In order to be compatible with the experimental features and for supporting language intelligence in VSCode Plugin, enable the `Allow Experimental` option in user settings.
+Ballerina Language Specification supports a set of experimental features such as *_transactions_* syntax. In order to be compatible with the experimental features and for supporting language intelligence in VSCode Plugin, enable the `Allow Experimental` option in user settings. And for Intellij Idea, goto `Preferences | Languages & Frameworks | Ballerina | Experimental Features` and tick the `Allow ballerina experimental features` check box.
```
Comment (praneesha) [inline website/two-column-pages/docs/learn/tools-ides.md]:
```suggestion
Ballerina Language Specification supports a set of experimental features such as the *_transactions_* syntax. In order to be compatible with the experimental features and for supporting language intelligence in VSCode Plugin, enable the `Allow Experimental` option in user settings. For IntelliJ IdEA, go to `Preferences | Languages & Frameworks | Ballerina | Experimental Features` and select `Allow ballerina experimental features.
```

File: website/two-column-pages/docs/learn/tools-ides.md [hunk 2]
```diff
@@ -42,4 +42,4 @@ You can test a Ballerina file or module using the following command:
 
 ## Compatibility
 
-Ballerina Language Specification supports a set of experimental features such as *_transactions_* syntax. In order to be compatible with the experimental features and for supporting language intelligence in VSCode Plugin, enable the `Allow Experimental` option in user settings.
+Ballerina Language Specification supports a set of experimental features such as the *_transactions_* syntax. In order to be compatible with the experimental features and for supporting language intelligence in VSCode Plugin, enable the `Allow Experimental` option in user settings. For IntelliJ IdEA, go to `Preferences | Languages & Frameworks | Ballerina | Experimental Features` and select `Allow ballerina experimental features`.
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn/tools-ides.md]: Does the IDE plugin say "Allow ballerina experimental features"?
Then it should be corrected there to say "Allow Ballerina experimental features". If not, here it should be updated as "ballerina" -> "Ballerina"

Comment (rdhananjaya) [reply website/two-column-pages/docs/learn/tools-ides.md]: Screen Shot 2019-09-09 at 3 02 12 PM

**Comment by MaryamZi** [comment]
The Tools and IDEs section is apparently being removed. These changes would have to go the page specific to the IDEA plugin. @NipunaRanasinghe @praneesha please note.

**Comment by praneesha** [comment]
@MaryamZi - Ack

**Comment by anupama-pathirage** [comment]
Closing this PR as per the about comment.

**Pull request closed without merging**